src/main/resources/db/migration/V1_1__offender_key_worker.sql

```diff
@@ -27,7 +27,7 @@ CREATE TABLE OFFENDER_KEY_WORKER
 
   CONSTRAINT OFFENDER_KEYWORKER_UNIQUE UNIQUE (OFFENDER_NO, STAFF_ID, ASSIGNED_DATE_TIME),
 
-  CONSTRAINT OFFENDER_KEY_WORKER_C1 CHECK (ALLOC_TYPE IN ('A','M'))
+  CONSTRAINT OFFENDER_KEY_WORKER_C1 CHECK (ALLOC_TYPE IN ('A','M','P'))
```

Comment:
@steverendell forgot to see this, you can't change flyway files - they are already in the db, and they will fail.  Hence why KW service is down.